src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/RedundantModifierCheck.java

```diff
@@ -60,6 +60,10 @@
  * {@code record} definitions that are declared as {@code final} and nested
  * {@code record} definitions that are declared as {@code static}.
  * </li>
+ * <li>
+ * {@code strictfp} modifier on java 17 or later. See reason at
```

Comment:
What constructs can this modifier be placed on? records, classes, interfaces, methods, abstract methods, ...?

Reply:
Any , https://github.com/checkstyle/checkstyle/blob/37dd5e815caf7b324cb15fd2da386ab299bc1ea4/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/InputRedundantModifierStrictfpWithJava17.java

That's why I kept it general on the doc and didn't mention specific constructs

Reply:
> Please add an example with this property.

I think your request will be blocked until https://github.com/checkstyle/checkstyle/pull/15376